roles/oVirt.image-template/library/ovirt_templates.py

```diff
@@ -0,0 +1,619 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# pylint: skip-file
```

Comment:
Why should pylint skip this file?

This file is generally lacking in comments and doc strings.

Reply:
CC @machacekondra 
I've made an attempt to fix much of the pylint, but as I'm not the module author, I'm stymied by the following issues:

'''
************* Module ovirt_templates
roles/oVirt.image-template/library/ovirt_templates.py:453: [R0914(too-many-locals), main] Too many local variables (24/20)
roles/oVirt.image-template/library/ovirt_templates.py:633: [W0703(broad-except), main] Catching too general exception Exception
roles/oVirt.image-template/library/ovirt_templates.py:453: [R0912(too-many-branches), main] Too many branches (19/12)
roles/oVirt.image-template/library/ovirt_templates.py:453: [R0915(too-many-statements), main] Too many statements (70/50)
error: lint error 12.
'''